app/components/nautilus/builder.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Nautilus
+  # A builder for articles that come from Nautilus
+  class Builder < GenericBuilder
+    private
+
+    def publication
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)